web/site/js/vis/PixiVisualizer.js

```diff
@@ -258,30 +352,36 @@ export default class PixiVisualizer extends VObject {
     }
 
     visSetNodePos(nodeId, x, y) {
-        this.nodes[nodeId].setPosition(x, y)
+        this.nodes[nodeId].setPosition(x, y);
+        this.logNode(nodeId, `Moved to (${x},${y})`)
     }
 
     visOnExtAddrChange(nodeId, extAddr) {
-        this.nodes[nodeId].extAddr = extAddr
+        this.nodes[nodeId].extAddr = extAddr;
+        this.logNode(nodeId, `Extended Address set to ${this.formatExtAddr(extAddr)}`)
     }
 
     visOnNodeFail(nodeId) {
-        this.nodes[nodeId].failed = true
+        this.nodes[nodeId].failed = true;
+        this.logNode(nodeId, "Radio is OFF")
     }
 
     visOnNodeRecover(nodeId) {
-        this.nodes[nodeId].failed = false
+        this.nodes[nodeId].failed = false;
+        this.logNode(nodeId, "Radio is ON")
     }
 
     visSetParent(nodeId, extAddr) {
-        this.nodes[nodeId].parent = extAddr
+        this.nodes[nodeId].parent = extAddr;
+        this.logNode(nodeId, `parent set to ${this.formatExtAddrPretty(extAddr)}`)
     }
 
     visSetTitle(title, x, y, fontSize) {
         this.titleText.text = title;
         this.titleText.x = x;
         this.titleText.y = y;
         this.titleText.style.fontSize = fontSize;
+        this.log(`Title set to "${title}", position (${x},${y}), font size ${fontSize}`)
```

Comment:
Can we also do `if (oldTitle != title && oldTitleX != x [...])`?

Reply:
Added checking for title text, but not x, y, and font size.